### ISSUE TITLE
Duplicate attributes - do they always cause a problem?

### ISSUE THREAD

**Issue by dd8** [auto-wcag/auto-wcag#352]
Not all attributes affect how content is presented to the user. Do these cause users a problem:

```
<meta name='generator' content='WordPress' content='Jekyll' />
<script defer defer> <!-- defer has no effect if src absent --> </script>
<a rel='nofollow' rel='nofollow'>External link</a> <!-- nofollow makes Googlebot ignore link -->
```

What happens when the attribute is duplicated with identical values? The browser can choose either as the true attribute value with no change in behaviour:

```
<img src='test.png' width='10' width='10' height='10' />
<img src='test.png' alt='Some alt text' alt='Some alt text' />

```

**Comment by dd8** [comment]
Also the HTML spec describes behaviour when duplicate attributes are present - browsers ignore duplicate attributes in HTML documents and use the first value:
https://www.w3.org/TR/html5/syntax.html#attribute-name-state

**Comment by ShadowBB** [comment]
I agree they do not always cause trouble for the user. But they do always fail WCAG. So the rules should definitely fail them. The argument presented here might be more suited for future WCAG discussions.

**Comment by dd8** [comment]
OK - I've filed an issue on WCAG 2.0 since it's in the normative text:
https://github.com/w3c/wcag/issues/542

**Comment by annethyme** [comment]
@dd8, would you be happy for us to close this issue until there are news from AGWG?

**Comment by dd8** [comment]
Yes, it will take months to resolve, and there are related problems for some other parsing errors (behaviour is defined in the HTML5 spec for some types of error)
